docs/source/interactive/autoawait.rst

```diff
@@ -131,23 +131,23 @@ a loop to run. By default IPython will use a fake coroutine runner which should
 allow ``IPython.embed()`` to be nested. Though this will prevent usage of the
 :magic:`%autoawait` feature when using IPython embed. 
 
-You can set explicitly a coroutine runner for ``embed()`` if you desire to run
-asynchronous code, the exact behavior is though undefined.
+You can set a coroutine runner explicitly for ``embed()`` if you want to run
+asynchronous code, though the exact behavior is undefined.
```

Comment:
I rephrased for flow but perhaps this is gratuitous. I am happy to change it back.

Reply:
Thanks ! I'm not a good english writer ! Much aprreciated.